Mathlib/ModelTheory/Semantics.lean

```diff
@@ -935,6 +935,20 @@ theorem realize_toFormula (φ : L.BoundedFormula α n) (v : Sum α (Fin n) → M
     · exact Fin.elim0 x
 #align first_order.language.bounded_formula.realize_to_formula FirstOrder.Language.BoundedFormula.realize_toFormula
 
+@[simp]
+theorem realize_iSup (s : Finset β) (f : β → L.BoundedFormula α n)
+    (v : α → M) (v' : Fin n → M) : (iSup s f).Realize v v' ↔
+    ∃ b ∈ s, (f b).Realize v v' := by
```

Comment:
Nitpick. I think it's a bit nicer to have the statement on its own line, instead of breaking after the `\iff`.
```suggestion
    (v : α → M) (v' : Fin n → M) :
    (iSup s f).Realize v v' ↔ ∃ b ∈ s, (f b).Realize v v' := by
```